ui_config/tenant/YJSG-Indore/app.json

```
@@ -107,6 +107,24 @@
 				"title": ""
 			}
 		},
+		{
+			"name": "createClassroom",
+			"path": "/create-classroom",
+			"component": "CreateClassroomForm",
+			"config": "classroomForm",
```

Rename this to createClassroomForm as there can be another classroomForm.
```suggestion
			"config": "createClassroomForm",
```